custom/skills.yaml

```diff
@@ -56,19 +56,13 @@ skills:
                 - [0.60, 0.0, 0.0, 0.0, 0.0]
                 - [0.45, 0.0, 0.0, 0.0, 0.0]
                 - [0.60, 0.0, 0.0, 0.0, 0.0]
-        left:
-            grasp_offset:  {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            marker_to_grippoint: {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-        left_suction:
-            grasp_offset:  {x: -0.0338, y: 0.012, z: 0.0844, roll: -1.57079632679, pitch: 0.87239795, yaw: 0.0}
-            marker_to_grippoint: {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-        right:
-            grasp_offset:  {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            marker_to_grippoint: {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-        right_suction:
-            grasp_offset:  {x: -0.0338, y: 0.012, z: 0.0844, roll: 0, pitch: 0.87239795, yaw: -1.57079632679}
-            marker_to_grippoint: {x: 0.0, y: 0.0, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
-            base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
+        base_offset: {x: 0.45, y: 0.08, z: 0.0, roll: 0.0, pitch: 0.0, yaw: 0.0}
+
+    gripper:
+        grasp_frame: 'grippoint_left'
```

Comment:
`arm_center` and `grippoint_left` are conflicting

Reply:
Agreed, will see if I can change the name of this frame